examples_tests/49.Radix_Sort/Vec2D.cpp

```diff
@@ -0,0 +1,9 @@
+﻿#include "Vec2D.h"
+
+
+nbl::Vec2D::Vec2D(const uint32_t Key, const uint32_t Data):
+	Key(Key),
+	Data(Data)
+{
```

Comment:
A) You'd need to add this file to CMake of the example 49
B) Such a function should be inline

Reply:
A) set(EXTRA_SOURCES Vec2D.cpp)  ?

Reply:
yes...

also dont put your own classes in the `nbl` namespace